fili-core/src/main/java/com/yahoo/bard/webservice/web/endpoints/EndpointServlet.java

```diff
@@ -46,43 +74,124 @@ public EndpointServlet(ObjectMappersSuite objectMappers) {
      */
     protected <T> Response formatResponse(
             ApiRequest apiRequest,
+            Response.ResponseBuilder builder,
+            Pagination pagination,
+            ContainerRequestContext containerRequestContext,
             Stream<T> rows,
             String jsonName,
             List<String> csvColumnNames
     ) {
+        UriInfo uriInfo = containerRequestContext.getUriInfo();
```

Comment:
Could we push `containerRequestContext.getUriInfo();` down to the only place where it is used?